perma_web/perma/templates/user_management/includes/created-link-items.html

```diff
@@ -1,8 +1,4 @@
-{% load local_datetime pipeline %}
-
-{% if in_iframe %}
-    {% stylesheet 'base' %}
-{% endif %}
```

Comment:
Aha. That sounds like the way to go. I'll add it to the list of potentially deletable code.